tools/pbprint.cc

```diff
@@ -3,110 +3,131 @@
 #include <iomanip>
 #include <iostream>
 #include <vector>
-#include <asio.hpp>
-#include <asio/serial_port.hpp>
-#include <asio/signal_set.hpp>
 #include <google/protobuf/io/coded_stream.h>
-#include "packet/frame.h"
+#include "cobs.h"
 #include "pose.pb.h"
 #include "simulation.pb.h"
+#include <asio.hpp>
+#include <asio/serial_port.hpp>
+#include <asio/signal_set.hpp>
```

Comment:
I have been using alphabetical order in other projects. Unfortunately sometimes the order is important in c++. We could do something like this when that is the case.

```c
// START ORDERED INCLUDES
#include "c.h"
#include "b.h"
#include "a.h"
// END ORDERED INCLUDES
#include "aa.h"
#include "bb.h"
```

Undid the moved imports.